hands_on/move_tags.py

```diff
@@ -0,0 +1,44 @@
+from exercise_utils.cli import run_command
+from exercise_utils.gitmastery import create_start_tag
+
+import os
+from pathlib import Path
+
+__requires_git__ = True
+__requires_github__ = True
+
+
+def download(verbose: bool):
+    """
+    hp-move-tags (T4L2)
+    - Fork https://github.com/git-mastery/samplerepo-preferences to user account
+    - Clone the fork locally as 'samplerepo-preferences'
+    - Create:
+        * lightweight tag `v1.0` on HEAD
+        * annotated tag  `v0.9` on HEAD~2 with message "First beta release"
+    """
+
+    # Ensure GitHub CLI is authenticated
+    gh_user = run_command(["gh", "api", "user", "--jq", ".login"], verbose).strip()
+    if not gh_user:
+        raise RuntimeError("GitHub CLI not authenticated. Run `gh auth login` and retry.")
+
+    upstream = "git-mastery/samplerepo-preferences"
+    target_dir = Path("samplerepo-preferences")
+
+    # Fresh sandbox, if re-downloading
+    if target_dir.exists():
+        import shutil
```

Comment:
I would avoid local imports, there's no issues with importing it at the start of the script